pkg/operator/ceph/controller/predicate.go

```diff
@@ -409,6 +409,31 @@ func WatchControllerPredicate() predicate.Funcs {
 					return true
 				}
 				logger.Debugf("no change in CR %q", objNew.Name)
+
+			case *cephv1.CephBlockPoolRadosNamespace:
+				objNew := e.ObjectNew.(*cephv1.CephBlockPoolRadosNamespace)
+				logger.Debug("update event on CephBlockPoolRadosNamespace CR")
```

Comment:
The rook operator can operate on multiple resources in multiple namespaces. All info/debug messages must list the namespaced name so context is clear.